readme.md

```diff
@@ -64,7 +67,8 @@ Your Answer:
 Describe the purpose of views and models in FE JS.
 
 Your Answer:
-```text
+```
+I believe models hold information, while views hold the behavior of the pages. The view is part of the app where the users interact with.
 
 ```
```

Comment:
Front-end models represent our data, and their actions. Generally, they contain methods for CRUD actions. In order to persist to the DB, a Front-End model will usually use AJAX requests to update the data accordingly on the back end.

Front-end views generally have two main purposes:
1. Take data (from F.E. Models) and render them to HTML and put them on the page (DOM).
2. Defining how to respond to user events (clicks, keypresses, etc), by updating the DOM, using models, etc.